src/js/WCartesianChart.js

```diff
@@ -512,8 +512,8 @@ WT_DECLARE_WT_MEMBER_BIG
         for (i = 0; i < len(sliders()); ++i) {
           var o = $('#' + sliders()[i]);
```

Comment:
I'd prefer it in the non-jQuery way here (we want to get rid of our jQuery dependency eventually):

```cpp
var o = document.getElementById(sliders()[i]);
```